doc/api/os.markdown

```diff
@@ -182,5 +182,16 @@ on OS X and `'Windows_NT'` on Windows.
 
 Returns the system uptime in seconds.
 
+## os.userInfo()
+
+Returns a subset of the password file entry for the current effective user. The
+returned object includes the `username`, `uid`, `gid`, `shell`, and `homedir`.
```

Comment:
`shell` and `homedir` should be swapped here because that's the way they appear in `passwd`.